src/globus_cli/login_manager/manager.py

```diff
@@ -313,6 +315,30 @@ def _get_gcs_info(
             raise ValueError("Internal Error! collection_id or endpoint_id is required")
         return (resolved_ep_id, epish)
 
+    def get_specific_flow_client(
+        self,
+        flow_id: uuid.UUID,
+    ) -> globus_sdk.SpecificFlowClient:
+        scope_name = f'flow_{str(flow_id).replace("-", "_")}_user'
+        self.add_requirement(
+            str(flow_id),
+            scopes=[f"https://auth.globus.org/scopes/{flow_id}/{scope_name}"],
+        )
```

Comment:
We should swap this out to the (recently corrected) SDK scope builder value.
I think it might even make sense to do
```python
client = SpecificFlowClient(flow_id)
self.add_requirement(str(flow_id), scopes=[client.scopes.user])
...
client.authorizer = ...
return client
```

The authorizer being created after-the-fact is a little unusual, but it doesn't violate any API/contract with the SDK.

Reply:
Argh, I missed this (and below). Thanks for catching these!